bindings/wasm/cypress/support/setup.js

```diff
@@ -1,9 +1,8 @@
-import * as client from "@iota/client-wasm/web";
+import init from "@iota/sdk-wasm/web";
```

Comment:
shouldn't this be `import { init } from ..` ? 🤔

Reply:
This is how it is done in https://github.com/iotaledger/iota-sdk/blob/develop/bindings/wasm/README.md#web, and it didn't work before when I used `import * as sdk ...` and then used `sdk.init`, so I'll not touch that again :joy:.